pkg/kubefedctl/enable/validation.go

```diff
@@ -107,6 +107,9 @@ func federatedTypeValidationSchema(templateSchema map[string]v1beta1.JSONSchemaP
 									Schema: &v1beta1.JSONSchemaProps{
 										Type: "object",
 										Properties: map[string]v1beta1.JSONSchemaProps{
+											"op": {
```

Comment:
Would it make sense to validate for the specific values that are supported?

Reply:
Should we consider making at least `path` and `value` fields required?

Reply:
Path can be required but value is required by `replace` and `add` but not `remove`.